### PR TITLE
[ChangeFeedProcessor]suppressE2ETimeoutConfigOnChangeFeedProcessor

### PR DESCRIPTION
**Pull request by xinlian12**
Disable cosmosEndToEndOperationLatencyPolicyConfig within ChangeFeedProcessor.

### CODE REVIEW

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @xinlian12

**Review by kushagraThapar — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #### Breaking Changes
 
 #### Bugs Fixed
+* Suppressed `CosmosEndToEndOperationLatencyPolicyConfig` in `ChangeFeedProcessor`. See [PR 36775](https://github.com/Azure/azure-sdk-for-java/pull/36775)
```
[the remark this answers is not in the document]
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md]: Updated. hmm, not sure whether it should go in to the Breaking change section, will discuss offline.

**Comment by xinlian12** [comment]
Failed Tests:
maxValueExceedingDefinedLimitStillWorksWithoutException -> passed locally. Failure not caused by this PR

**Pull request merged**